NebulaModel/Packets/PacketProcessor.cs

```diff
@@ -1,15 +1,13 @@
-﻿using NebulaModel.Networking;
+﻿using NebulaAPI;
+using NebulaModel.Networking;
 
 namespace NebulaModel.Packets
 {
-    public abstract class PacketProcessor<T>
+    public abstract class PacketProcessor<T> : BasePacketProcessor<T>
     {
-        protected bool IsHost;
-        protected bool IsClient => !IsHost;
-
-        internal void Initialize(bool isHost)
+        public override void ProcessPacket(T packet, INebulaConnection conn)
         {
-            IsHost = isHost;
+            ProcessPacket(packet, (NebulaConnection)conn);
```

Comment:
Which "ProcessPacket" is this calling? I think prefixing this with the relevent class would be better here (ie, `base.` or `this.`) to better disambiguate

Reply:
I think cast to NebulaConnection disambiguates it. Plus both really belong to this.